src/logic/tokens/store/actions/fetchSafeTokens.js

```diff
@@ -78,8 +77,14 @@ const fetchSafeTokens = (safeAddress: string) => async (dispatch: ReduxDispatch<
     const updateActiveTokens = activeTokens.equals(alreadyActiveTokens) ? noFunc : update({ activeTokens })
     const updateBalances = balances.equals(safeBalances) ? noFunc : update({ balances })
     const updateEthBalance = ethBalance === currentEthBalance ? noFunc : update({ ethBalance })
-
-    const updateCurrencies = currencyList.equals(storedCurrencyBalances) ? noFunc : setCurrencyBalances(currencyList)
+    const storedCurrencyBalances =
```

Comment:
see, there we have `storedCurrencyBalances` instead of `currencyBalancesStored`, please watch for the naming to be consistent :)